apps/game_client/lib/game_client/protobuf/messages.pb.ex

```diff
@@ -6,6 +6,7 @@ defmodule GameClient.Protobuf.GameStatus do
   field(:PREPARING, 0)
   field(:RUNNING, 1)
   field(:ENDED, 2)
+  field(:PICKING_BOUNTY, 3)
```

Comment:
Not mandatory but `SELECTING` or `CHOOSING` sounds more accurate to me.

Reply:
Totally!